backend/shop/serializers.py

```diff
@@ -30,6 +30,16 @@ def get_full_name(self, obj):
         return f"{obj.first_name} {obj.last_name}"
 
 
+class ShopProfileSerializerPlusGroups(ShopProfileSerializer):
+    groups = SerializerMethodField()
+
+    class Meta(ShopProfileSerializer.Meta):
+        fields = ShopProfileSerializer.Meta.fields + ["groups"]
```

Comment:
```suggestion
    class Meta:
        fields = ..
```

Don't do what you're doing above. Have just `Meta` and don't do abstractions, it's going to be hard to spot bugs.

Reply:
Do you suggest, that I rewrite all fields from the current parent?
But then If the fields are changed, they must be corrected in several places